src/main/java/tutorial/AddMessage.java

```diff
@@ -0,0 +1,16 @@
+//package tutorial;
```

Comment:
Is there a reason these are all commented out?

Reply:
just so i can run the UI files to check the results, because addmessage and DM files contains errors